### ISSUE TITLE
The loss did not decrease during training, please check the code

### ISSUE THREAD

**Issue by WangBingJian233** [YazhouZhu19/RPT#3]
Hello author, thank you very much for your excellent work. But I obviously get an error based on the drop in loss when your code runs, can you check the code? There are also papers that mention that you used cross-entropy loss, boundary loss, DICE loss, and your code lacks DICE loss, and the boundary loss has not decreased during training. Looking forward to your code update.

**Comment by YazhouZhu19** [comment]
Hello, thank you very much for pointing out errors in code. I have checked and update the codes.

**Comment by WangBingJian233** [comment]
Dear author, thank you for your reply. When I run your updated code, I find that bd_loss and dice_loss have not decreased during the training process (query_loss is a normal decline), is this situation normal?

**Comment by YazhouZhu19** [comment]
In training stage, due to the unbalance of segmentation regions provided by supervoxel based pseudo labels, bd_loss and dice_loss will fluctuate. Take the focal loss may be better?
